<<<<<<< HEAD
CRITICAL - 2020-12-02 01:51:31 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`per...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(35): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(31): App\Models\Escolares\ActividadModel->get_actividades_alumno_calificadas('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(30): App\Services\Escolares\ActividadService->actividades_alumno_calificadas('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 11:19:24 --> Trying to get property 'usuario' of non-object
#0 C:\xampp\htdocs\comple\app\Views\Escolares\navbar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 7, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Escolares/navba...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(17): view('Escolares/navba...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 12:00:26 --> syntax error, unexpected '}', expecting end of file
#0 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 12:02:19 --> syntax error, unexpected '}', expecting end of file
#0 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 12:05:43 --> syntax error, unexpected '}', expecting end of file
#0 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 12:31:23 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(26): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 12:31:23 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Fallo al parsear la cadena JSON, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\comple\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp\htdocs\comple\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\comple\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 12:35:58 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(26): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 12:35:58 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Fallo al parsear la cadena JSON, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\comple\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp\htdocs\comple\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\comple\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 16:36:36 --> Trying to get property 'usuario' of non-object
#0 C:\xampp\htdocs\comple\app\Views\Escolares\navbar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 7, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Escolares/navba...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(17): view('Escolares/navba...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 16:40:26 --> Undefined property: App\Controllers\Escolares\GenerarController::$actividadService
#0 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(23): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 23, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 16:40:28 --> Undefined property: App\Controllers\Escolares\GenerarController::$actividadService
#0 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(23): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 23, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 16:40:33 --> Undefined property: App\Controllers\Escolares\GenerarController::$actividadService
#0 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(23): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 23, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 16:41:28 --> Table 'acitp_complementarias.alumnos' doesn't exist
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Escolares\GenerarService.php(16): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl('161160176')
#6 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(23): App\Services\Escolares\GenerarService->getAlumnoPorNoControl('161160176')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-02 16:41:32 --> Table 'acitp_complementarias.alumnos' doesn't exist
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Services\Escolares\GenerarService.php(16): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl('161160176')
#6 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(23): App\Services\Escolares\GenerarService->getAlumnoPorNoControl('161160176')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-02 16:43:13 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\GenerarService.php(16): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(23): App\Services\Escolares\GenerarService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 16:48:54 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(29): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 16:48:54 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Fallo al parsear la cadena JSON, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\comple\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp\htdocs\comple\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\comple\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 16:49:00 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(29): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 16:49:00 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Fallo al parsear la cadena JSON, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\comple\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp\htdocs\comple\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\comple\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 17:48:16 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(33): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 17:48:16 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Fallo al parsear la cadena JSON, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\comple\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp\htdocs\comple\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\comple\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 17:49:20 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(33): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 17:49:20 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Fallo al parsear la cadena JSON, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\comple\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp\htdocs\comple\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\comple\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 17:50:55 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(33): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 17:50:55 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Fallo al parsear la cadena JSON, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\comple\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp\htdocs\comple\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\comple\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 17:59:52 --> Undefined variable: alumno
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(33): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 17:59:52 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Fallo al parsear la cadena JSON, error: "Malformed UTF-8 characters, possibly incorrectly encoded". in C:\xampp\htdocs\comple\system\Format\Exceptions\FormatException.php:9
Stack trace:
#0 C:\xampp\htdocs\comple\system\Format\JSONFormatter.php(71): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(414): CodeIgniter\Format\JSONFormatter->format(Array)
#2 C:\xampp\htdocs\comple\system\API\ResponseTrait.php(134): CodeIgniter\Debug\Exceptions->format(Array)
#3 C:\xampp\htdocs\comple\system\Debug\Exceptions.php(168): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 19:59:05 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\GenerarModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(21): App\Models\Escolares\GenerarModel->getAlumnoPorNoControl(NULL)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constancia()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 20:04:04 --> Undefined variable: alumnos
#0 C:\xampp\htdocs\comple\app\Views\Escolares\Generar\listar.php(30): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 30, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('escolares/gener...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(31): view('escolares/gener...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constancia()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 20:05:56 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`per...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(35): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(31): App\Models\Escolares\ActividadModel->get_actividades_alumno_calificadas('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(30): App\Services\Escolares\ActividadService->actividades_alumno_calificadas('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 20:06:11 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`per...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(35): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(31): App\Models\Escolares\ActividadModel->get_actividades_alumno_calificadas('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(30): App\Services\Escolares\ActividadService->actividades_alumno_calificadas('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 20:35:45 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(19): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(16): App\Models\Escolares\ActividadModel->getAlumnoPorNoControl('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(29): App\Services\Escolares\ActividadService->getAlumnoPorNoControl('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 20:36:06 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`per...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Escolares\ActividadModel.php(35): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Escolares\ActividadService.php(31): App\Models\Escolares\ActividadModel->get_actividades_alumno_calificadas('161160176')
#4 C:\xampp\htdocs\comple\app\Controllers\Escolares\ActividadController.php(30): App\Services\Escolares\ActividadService->actividades_alumno_calificadas('161160176')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\ActividadController->buscar()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\ActividadController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 23:22:14 --> Undefined variable: no_control
#0 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(37): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 37, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constancia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 23:22:34 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 266342400 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-02 23:22:41 --> Undefined variable: no_control
#0 C:\xampp\htdocs\comple\app\Controllers\Escolares\GenerarController.php(37): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 37, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Escolares\GenerarController->constancia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Escolares\GenerarController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 23:23:02 --> Allowed memory size of 536870912 bytes exhausted (tried to allocate 264245248 bytes)
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
=======
CRITICAL - 2020-12-02 00:56:11 --> Trying to get property 'rfc_responsable' of non-object
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(17): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 17, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 01:50:14 --> Undefined variable: periodo
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 01:51:02 --> Undefined variable: periodo
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 01:51:44 --> Undefined variable: periodo
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 01:51:58 --> Undefined variable: periodos
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 01:52:31 --> Undefined variable: periodos
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 01:53:00 --> Undefined variable: periodos
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 01:53:54 --> Undefined variable: periodos
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 01:54:09 --> Undefined variable: descripcion
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 02:07:26 --> Undefined variable: id_actividadd
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 02:10:59 --> Undefined variable: periodo
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 24, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-12-02 02:18:01 --> Unknown column 'p.perio' in 'field list'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `a`.`id_...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `a`.`id_...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `a`.`id_...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`id_...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(38): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(22): App\Models\Responsable\AsistenciaModel->get_actividad('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-12-02 17:03:15 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(29): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(60): App\Models\Responsable\AsistenciaModel->get_alumno('201160132')
#4 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(21): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 17:10:10 --> MySQL server has gone away
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(29): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(60): App\Models\Responsable\AsistenciaModel->get_alumno('201160061')
#6 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(21): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('11')
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#9 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2020-12-02 18:46:42 --> syntax error, unexpected 'public' (T_PUBLIC)
#0 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 19:03:22 --> Use of undefined constant count - assumed 'count' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(52): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): App\Models\Responsable\AsistenciaModel->get_inscripcion('11')
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-02 19:05:07 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(66): App\Models\Responsable\AsistenciaModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(21): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 19:05:20 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(66): App\Models\Responsable\AsistenciaModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(21): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 19:05:47 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(66): App\Models\Responsable\AsistenciaModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(21): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-12-02 19:06:23 --> Unknown column 'count' in 'field list'
#0 C:\xampp\htdocs\comple\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT `count`,...')
#1 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `count`,...')
#2 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `count`,...')
#3 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `count`,...', Array, false)
#4 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(54): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): App\Models\Responsable\AsistenciaModel->get_inscripcion('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-12-02 19:07:02 --> Use of undefined constant count - assumed 'count' (this will throw an Error in a future version of PHP)
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(52): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): App\Models\Responsable\AsistenciaModel->get_inscripcion('11')
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-12-02 19:22:03 --> Undefined variable: contar
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Asistencia\listar.php(75): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 75, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(38): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-12-02 19:22:45 --> Undefined variable: c
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Asistencia\listar.php(77): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 77, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(38): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
>>>>>>> b87f9bc5167f825e8ce6bb4e8377cc4822b91d1f
